scripts.d/s-salome-openturns

```diff
@@ -17,44 +17,61 @@ function s-salome-openturns-tar()
     echo OPENTURNS_SRC_$VERSION.tgz
 }
 
-function s-salome-openturns-build-depends()
+function s-salome-openturns-common-build-depends()
+{
+    echo s-cmake
+}
+
+function s-salome-openturns-default-build-depends()
 {
     echo os@-python3-sphinx
     echo os@-swig
     echo os@-graphviz-dev
     echo os@-doxygen
     echo os@-python3-docutils
+}
 
-    echo s-cmake
+function s-salome-openturns-common-depends()
+{
     echo s-salome-configuration
+    echo s-persalys
 }
 
-function s-salome-openturns-depends()
+function s-salome-openturns-default-depends()
 {
     echo os@-qt5-default
     echo os@-qtbase5-dev
 
-    echo s-persalys
     echo s-salome-kernel
+    echo s-salome-gui
     echo s-openturns
     echo s-yacs
 }
 
+function s-salome-openturns-linux-depends()
+{
+    echo s-salome-bin
+}
+
 function s-salome-openturns-env()
 {
     local PREFIX=$1
     local TARGET=$2
 
-    # Use system Qt5
-    set-var    QT5_ROOT_DIR       /usr
-
     set-var    OPENTURNS_ROOT_DIR $PREFIX
 
     add-to-var LD_LIBRARY_PATH    $PREFIX/lib/salome
     add-to-var PYTHONPATH         \
                $PREFIX/lib/python$PYTHON_VERSION/site-packages/salome
 }
 
+function s-salome-openturns-test()
```

Comment:
Yes a good idea to use a hook to get the actual location of the test directory. I just think that we should rename `-test` to `-ctest-dir` to avoid any confusion with the scbi default `-tests` hook for running the testsuite (if any).

Reply:
i have modified test into ctest-dir.  :+1: